src/test/java/core/basesyntax/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyTest {
+    private static OperationStrategy operationStrategy;
+    private static OperationHandler expectedOperationHandler;
+
+    @BeforeEach
+    void setUp() {
+        Map<FruitTransaction.Operation, OperationHandler> operationsMap = new HashMap<>();
+        operationsMap.put(FruitTransaction.Operation.BALANCE, new BalanceOperationHandler());
+        operationsMap.put(FruitTransaction.Operation.SUPPLY, new SupplyOperationHandler());
+        operationsMap.put(FruitTransaction.Operation.PURCHASE, new PurchaseOperationHandler());
+        operationsMap.put(FruitTransaction.Operation.RETURN, new ReturnOperationHandler());
+        operationStrategy = new OperationStrategy(operationsMap);
+    }
+
+    @Test
+    public void get_getBalanceHandler_Ok() {
```

Comment:
fix each method in OperationStrategyTest
```suggestion
    public void getBalanceHandler_Ok() {
```